keyboards/sol/config.h

```diff
@@ -20,3 +20,75 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
PSM is depreciated, as it's the default behavior. Please remove. 

```suggestion
```

Reply:
Fixed